libvips/resample/thumbnail.c

```diff
@@ -827,9 +851,10 @@ vips_thumbnail_build( VipsObject *object )
 			in = t[7];
 		}
 	}
-	else if( thumbnail->export_profile &&
-		(vips_image_get_typeof( in, VIPS_META_ICC_NAME ) || 
-		 thumbnail->import_profile) ) {
+	else if( thumbnail->export_profile ) {
```

Comment:
Do you think this works when someone does:
```bash
$ vipsthumbnail --export-profile=srgb x.jpg
````
where `x.jpg` is an image _without_ embedded ICC profile? I think the previous check was there to handle this case (perhaps as a speed improvement).

Reply:
I think this is OK -- `have_imported` is now only set TRUE if `vips_image_get_typeof( in, VIPS_META_ICC_NAME ) || thumbnail->import_profile` earlier in the function, and we only hit this if if `!have_imported`.

Reply:
But you're right, this won't work if `k2.jpg` has no profile and perhaps it should:

```
$ vipsthumbnail k2.jpg --export-profile cmyk
vipsthumbnail: unable to thumbnail k2.jpg
icc_transform: no input profile
```

It should assume that `k2.jpg` is in srgb.

Reply:
I committed a thing for this case.